### PR TITLE
chore(deck): Get devtools working with Deck

### PR DESCRIPTION
**Pull request by ZaymonFC**

### CODE REVIEW

File: packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx [hunk 1]
```diff
@@ -199,48 +202,62 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
...
+          const { active, object } = data;
+
           switch (role) {
             case 'settings':
               return data.plugin === meta.id ? <DebugSettings settings={settings.values} /> : null;
             case 'status':
               return <DebugStatus />;
           }
 
-          if (!settings.values.debug) {
```
Comment (ZaymonFC) [inline packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx:210]: `settings.values.debug` and `settings.values.devtools` can be enabled separately.

File: packages/sdk/app-framework/src/plugins/common/navigation.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import type { IntentData } from '../IntentPlugin';
 import type { Plugin } from '../PluginHost';
 
 // NOTE(thure): These are chosen from RFC 1738’s `safe` characters: http://www.faqs.org/rfcs/rfc1738.html
-export const SLUG_LIST_SEPARATOR = '.';
+export const SLUG_LIST_SEPARATOR = '+';
```
Comment (ZaymonFC) [inline packages/sdk/app-framework/src/plugins/common/navigation.ts:11]: + gives nicer URL semantics and allows plank id's to contain `.` like in the case of `dxos.org.plugin.debug.devtools`.

Comment (wittjosiah) [reply packages/sdk/app-framework/src/plugins/common/navigation.ts:11]: @thure

**Review by wittjosiah — APPROVED** [review]

File: packages/sdk/app-framework/src/plugins/common/navigation.ts (hunk 2 above)
Comment (thure) [reply packages/sdk/app-framework/src/plugins/common/navigation.ts:11]: I'd be fine with this change, sure 👍

**Pull request merged**